app/controllers/reviewers/applications_controller.rb

```diff
@@ -5,6 +5,7 @@ module Reviewers
   class ApplicationsController < Reviewers::BaseController
     before_action :store_filters, only: :index
     before_action :persist_order, only: :index
+    helper_method :mentor_comments
```

Comment:
I personally really dislike the `helper_method` thing... Aside personal feelings: it would make sense to have this available only on the `show` action or at least actions where you'd also have an `@application` instance var available.

Reply:
Hmm, dunno if I'm reading this wrong, but it seems that there's no way to make this available to specific actions?
http://api.rubyonrails.org/classes/AbstractController/Helpers/ClassMethods.html

Reply:
Oh ok, then no worries 💚